packages/core/src/components/axes/ruler.ts

```diff
@@ -30,8 +30,10 @@ export class Ruler extends Component {
 		const ruler = DOMUtils.appendOrSelect(svg, "g.ruler").attr("opacity", 1);
 		const line = DOMUtils.appendOrSelect(ruler, "line.ruler-line");
 		const dataPoints = svg.selectAll("[role=graphics-symbol]");
+		const height = Number(this.backdrop.attr("height"));
```

Comment:
We should use `DOMUtils.getSVGElementSize()` if this is something that might change since some browsers will mess up that calculation